pkg/migration/controllers/resourcetransformation.go

```diff
@@ -121,6 +126,9 @@ func (r *ResourceTransformationController) handle(ctx context.Context, transform
 			return nil
```

Comment:
TODO: During r.validateSpecPath(transform) need to remove the filter getsupportsk8stypes to allow cr and crds in rt only for the restore flow